s3cortxutils/s3confstore/s3confstore/cortx_s3_confstore.py

```diff
@@ -225,6 +245,13 @@ def run(self):
       else:
         sys.exit("Failed to read private ip from confstore of machineid: {}".format(args.machineid))
 
+    elif args.command == 'getpublicip':
```

Comment:
You need to add "getpublicip" to the 'args' first. Check above, line: 202

Also please test the code (both help and the option you are adding) should work fine. It seems you have not tested the "s3confstore binary"  !

Reply:
Addressed this and tested locally :
`[root@ssc-vm-2053 s3confstore]# s3confstore "json:///root/s3confstore.json" getpublicip --machineid 30512e5ae6df9f1ea02327bab45e499d
127.0.0.1
`